distribution/docs/src/main/resources/content/_configuring/map-layers-intrigue.adoc

```diff
@@ -0,0 +1,39 @@
+:title: Configuring Map Layers for ${catalog-ui}
+:type: subConfiguration
+:status: published
+:parent: Configuring ${catalog-ui}
+:order: 00
+:summary: Configuring map layers for ${catalog-ui}.
+
+== {title}
+
+Customize the look of the map displayed to users in ${catalog-ui} by adding or removing map layers through the *Map Layers UI*.
+
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-catalog}* application.
+. Select the *Map Layers* tab.
+. *Add*, *Configure* or *Remove* map layers as desired.
+
+.Adding a Map Layer
```

Comment:
We may want to point out that different map layers require different parameters.  For example `WMTS` may require a set of `tileMatrixLabels`.  See https://github.com/codice/ddf/pull/1763 for an example.